### PR TITLE
Fixes#1788, #1784, Remove jQuery scroll animation

### LINKED ISSUE #1788: Fix imports for Terms.react.js, Devices.react.js
**Actual Behaviour**

Fix imports

**Expected Behaviour**

Order them properly

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #1788, #1784 

Changes: 

1. Removed jQuery animation and added vanilla js animation
2. Fixed imports for Terms.react.js and Devices.react.js, and remove redundant document.style property

Demo Link: https://pr-1784-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by AakashMallik** [comment]
@fragm3 Just to keep things more flexible, add another arg to your scrollToTop function. "Fast", "Medium", "Slow".

**Comment by fragm3** [comment]
@AakashMallik third arg does allow user to add duration

**Comment by AakashMallik** [comment]
Oh, I was hoping for some preset values for these keywords as we would be using your function directly. But I guess that works fine as well. :innocent:

File: src/utils/animateScroll.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+export const scrollToTopAnimation = (
```
Comment (akshatnitd) [inline src/utils/animateScroll.js]: Good that you made it a utils :v:

**Pull request merged**